setup.py

```diff
@@ -68,10 +68,9 @@
             "six>=1.10.0",
             "lifelines>=0.9.1.0",
             "scikit-learn>=0.17.1",
-            "isovar==0.0.3",
         ],
         dependency_links=[
-            "https://github.com/hammerlab/isovar/archive/v0.0.3.tar.gz#egg=isovar-0.0.3",
+            "git+git://github.com/hammerlab/isovar",
```

Comment:
Alex bumps the `isovar` version on every commit, and this fails whenever he does. That seems problematic, so I went back to the non-version-specific setup.